R/experiment.R

```diff
@@ -68,22 +71,42 @@ new_experiment <- function(parameters, obsrates, tmax, seed, experiment, model,
     tmax <- as.integer(tmax)
   }
 
-# returning the object:
-  structure(setNames(cbind(parameters,
+  obsrates[] <- lapply(obsrates, as.integer)
+
+  model_info <- list("path" = model,
+                     "info" = read_gaml_experiment(experiment, model),
+                     "md5sum" = md5sum(model))
+  out <- structure(setNames(cbind(parameters,
                            obsrates,
                            tmax = as.integer(tmax),
                            seed = seed,
                            output = output), c(newnames, "tmax", "seed", "output")),
             class      = c("experiment", "tbl_df", "tbl", "data.frame"),
-            model      = model,
+            model      = model_info,
```

Comment:
I prefer to keep 'model_info' for 'model' attribute. When the server is ready, effectively we'll change the interface to 'model = model' or 'model = path' because we use the variable 'model' for path th gaml file. Having 2 attributes 'model' and 'model_info' is confusing.

Reply:
I agree if it'a easier to remove it when the server is ready.